ecs-cli/modules/cli/local/converter/marshal.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2015-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package converter
+
+import (
+	composeV3 "github.com/docker/cli/cli/compose/types"
+	"gopkg.in/yaml.v2"
+)
+
+// MarshalComposeConfig serializes a Docker Compose object into a YAML document.
```

Comment:
Is it accurate to say the out put is a YAML document? I think I still think of the out put as a payload that we then choose to write as a YAML document, but not sure if that's splitting hairs.

Reply:
I actually took this from the yaml package docs 😝
> Marshal serializes the value provided into a YAML document. 

https://godoc.org/gopkg.in/yaml.v2#Marshal